index.md

```diff
@@ -167,14 +167,21 @@ This is an overview of all Android versions and their corresponding identifiers
     {% include progress-cell.html rowspan=1 percentage=97.0 %}
   </tr>
   <tr>
-    <td>
+    <td rowspan="2">
       <b><a href="https://developer.android.com/about/versions/marshmallow">Android 6</a></b>
     </td>
     <td>Level 23</td>
     <td><code>M</code></td>
     <td>Marshmallow</td>
     {% include progress-cell.html rowspan=1 percentage=98.4 %}
-    <td rowspan="2">2015</td>
+    <td rowspan="3">2015</td>
+  </tr>
+  <tr class="table-notes">
+    <td colspan="3">
+        <ul>
+          <li>Google Play services (July 2024) <a href="https://support.google.com/googleplay/answer/9037938?hl=en">drops</a> <a href="https://9to5google.com/2024/07/11/android-lollipop-google-play-services/">support</a> for API levels below 23.</li>
```

Comment:
Strange without version number, do you know if it's mentioned in release notes somewhere?
Blog doesn't have it: https://android-developers.googleblog.com/search/label/Google%20Play%20services?m=1&max-results=12

Reply:
unfortunately, I did not find a version number.. (but I'm no expert)

it is also mentioned on this page: https://developers.google.com/android/guides/overview#services

Reply:
Interesting, these are all I found from release notes too.

https://developers.google.com/android/guides/releases#may_29_2024

https://firebase.google.com/support/release-notes/android#2024-05-02

Possible that only Auth needs it yet, but Auth is used by everything?

Reply:
@ebelinski I think this is mergeable as is without an official blog post or release notes... If there's one hopefully someone will update it :)

Reply:
it is clearly indicated in the google play support article, but maybe the docs are ahead of the release effectively dropping support?? :) just guessing..

Reply:
That might be a first 🫣